addons/common/functions/fnc_getFirstObjectIntersection.sqf

```diff
@@ -1,7 +1,7 @@
 #include "script_component.hpp"
 /*
  * Author: Ruthberg
- * Returns the the first intersection with terrain between two positions. @todo rewrite using lineIntersectsSurfaces?
+ * Returns the the first intersection with terrain between two positions.
```

Comment:
Binary seach isn't needed anymore and this func could just be rewritten with
```
params ["_source", "_destination"];

private _result = [false, [0, 0, 0]];
private _hits = lineIntersectsSurfaces [_source, _destination, objNull, objNull, true, -1];
{
    _x params ["_pos", "", "_obj"];
    if (!isNull _obj) exitWith {
        _result = [true, _pos];
    };
} forEach _hits;
_result
```

Reply:
Replaced with the function above